### PR TITLE
PAL-855 streaming architecture quickstart

### PR DESCRIPTION
**Pull request by dev930018**
~Requires all repos to be tagged with `palisade-0.5.0` once happy with the state of 0.5.0 release~
Currently uses `develop`, a future update may freeze this to an 0.5.0 tag

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
 <!---
-Copyright 2020 Crown Copyright
+Copyright 2018-2021 Crown Copyright
```
Comment (m78233) [inline QUICKSTART.md:2]: Had a look and there are about 1/2 dozen with only the 2020.  All are md files.

**Review by m78233 — CHANGES_REQUESTED** [review]
I found a few Copyright headers that needed updating.  All were md files.  The rest seems ok.

File: QUICKSTART.md (hunk 1 above)
Comment (dev930018) [reply QUICKSTART.md:2]: Won't do as out-of-scope.
Better addressed when the documentation of the repo is updated in a future ticket

**Review by m78233 — DISMISSED** [review]

File: QUICKSTART.md [hunk 2]
```diff
@@ -39,8 +40,8 @@ Run the cross-platform [quickstart.cmd script](quickstart.cmd):
   ```
 
 This will perform the following tasks necessary to set-up and start using Palisade:
-* Download each of the Palisade repos required to run the example ([common](https://https://github.com/gchq/Palisade-common), [readers](https://github.com/gchq/Palisade-readers), [clients](https://github.com/gchq/Palisade-clients), [services](https://github.com/gchq/Palisade-services), [examples](https://github.com/gchq/Palisade-examples))
-    - Since Palisade remains in active development, we will be pulling the 0.4.0 release which uses a REST-based microservice architecture
+* Download each of the Palisade repos required to run the example ([common](https://github.com/gchq/Palisade-common/tree/develop), [readers](https://github.com/gchq/Palisade-readers/tree/develop), [clients](https://github.com/gchq/Palisade-clients/tree/develop), [services](https://github.com/gchq/Palisade-services/tree/develop), [examples](https://github.com/gchq/Palisade-examples/tree/develop))
+    - Since Palisade remains in active development, we will be pulling the 0.5.0 release which uses a REST-based microservice architecture
```
Comment (developer01189998819991197253) [inline QUICKSTART.md:44]: No longer a rest based microservice

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**